test/jdk/com/sun/net/httpserver/Test1.java

```diff
@@ -138,34 +153,12 @@ public boolean verify (String s, SSLSession s1) {
         is.close();
         fout.close();
 
-        if (count != size) {
+        if (count != filePath.toFile().length()) {
             throw new RuntimeException ("wrong amount of data returned");
         }
-        String orig = root + "/" + f;
-        compare (new File(orig), temp);
+        Path tempPath = temp.toPath();
+        assert Files.mismatch(filePath, tempPath) < 0;
```

Comment:
Same suggestions WRT assertEquals here

Reply:
Fixed in 9c7c6af013e0495b08bcd248e30d83385bcc35b5.